### ISSUE TITLE
Archive repo?

### ISSUE THREAD

**Issue by Ms2ger** [WebAssembly/reference-types#140]
I got confused for a moment because this repo doesn't acknowledge that it's been merged into the main spec.

**Comment by rossberg** [comment]
Good point, I'll archive it. I don't recall that we have any guidelines regarding this. Should perhaps be added to the howto?
